### ISSUE TITLE
Integrating SPQR into existing code-first GraphQL app.

### ISSUE THREAD

**Issue by dnebing** [leangen/graphql-spqr#184]
I have an app that I started that is code-first but completely manual.  Has all of the builders, etc for all of the queries and mutations.  Following the graphql-java-servlet's OsgiGraphQLHttpServlet model, they are structured as "providers" which can provide types that, when building the schema, will be included.

In order to integrate SPQR into the mix, I somehow need a way to be able to similarly support a provider pattern so my existing queries/mutations/subscriptions can be included in the schema build.

I thought the GraphQLSchemaProcessor was my door to inject my existing objects into the GraphQLSchema.Builder instance before the final schema build. but that is an opaque object that does not support modification of the in-flight queries, mutations and subscriptions that have previously been added.

So basically I need a way to allow external components to provide or contribute queries, mutations and/or subscriptions into the process owning the GraphQLSchema.Builder schema building.
